projects/storefrontlib/layout/main/storefront.component.ts

```diff
@@ -120,8 +120,9 @@ export class StorefrontComponent implements OnInit, OnDestroy {
   }
 
   protected focusOnFirstNavigationItem() {
-    const closestNavigationUi =
-      this.elementRef.nativeElement.querySelector('cx-navigation-ui');
+    const closestNavigationUi = this.elementRef.nativeElement.querySelector(
+      '.NavigationBar cx-navigation-ui'
```

Comment:
Following up on your comment there are not 2 `cx-navigation-ui` elements on the page, but 3 if use is logged in
<img width="1253" alt="image" src="https://github.com/user-attachments/assets/e3e80ab4-c950-4913-8ac6-a158a9ef8029">

The intention of my changes was to find cx-navigation-ui which is the first on the page: either from `.NavigationBar` or the "My Account". The one in footer should still be the last, at least that was my assumption. So the solution which you've proposed will result in the situation which we had before: if the user is logged in, the focus still lands on the first category menu instead of the my account.

Reply:
As far as I understand your issue, after opening the hamburger menu focus landed on the first footer item, which shouldn't happen if there are any `cx-navigation-ui` higher in the DOM, because `document.querySelector` returns first element that matched the selector. Can you please send me a link to the environment where I can reproduce the issue?